main.cpp

```diff
@@ -311,8 +457,25 @@ struct Dictaphone
     void exportAudioFile(float audioData);
     void recordingOn(bool activeRecording);
     std::string displayAvailableSpace(Dictaphone dictaphoneStorage);
+
+    // new function
+    int flagNumberOfSpeakers();
 };
 
+int Dictaphone::flagNumberOfSpeakers()
+{    
+    int Voices = 4;
```

Comment:
Fixme: variable name does not conform to course coding standard
- [ ] revise

Reply:
Ah yes! That slipped through